lib/services/v2.0/nomination_pools/txs/chill.dart

```diff
@@ -1,7 +1,6 @@
 import 'package:karma_coin/services/v2.0/nomination_pools/types.dart';
 import 'package:karma_coin/services/v2.0/txs/tx.dart';
 
-/// @HolyGrease please add code doc for this tx - what is it used for? what is the use case? what does chlling mean?
```

Comment:
From substrate [documentation](https://github.com/paritytech/substrate/blob/master/frame/staking/README.md#chilling):
Finally, any of the roles above can choose to step back temporarily and just chill for a while.
This means that if they are a nominator, they will not be considered as voters anymore and if
they are validators, they will no longer be a candidate for the next election.